lib/boolean.ts

```diff
@@ -0,0 +1,54 @@
+export function boolean(value: unknown): boolean {
```

Comment:
I think we can add a comment at the top about where these are originally from:

```suggestion
// Originally from: https://www.npmjs.com/package/boolean

export function boolean(value: unknown): boolean {
```

Reply:
I added in d70dda4.